loomi/regulariza-morador-client
# Refactor/fix research flow 2

## Feature
  - update questions json
  - questions inputs with validation
  - add auxiliarImage
---

**What I did:**
- update questions json
 - add all missing properties in each question object
 - add validation in the answers ( CPF, CNPJ, required)
 - add dependent mark navigation
 - fix responsivity issues in the question page
 - add mask in date text input
 - add auxiliarImage in some questions

**How to test:**

1. In your browser go to [stage](http://project.netlify.app/) or [localhost](http://localhost:3000/)
2. Check if the component/view looks just like the [mockup](https://xd.adobe.com/) in different screen sizes (width between 1024px and 1920px)
3. Test the things that you think are worth testing, even the ones that are not in this description

![image](https://user-images.githubusercontent.com/32523401/134615341-c0bb8da7-6046-4083-9713-744bf2ee5eef.png)
![image](https://user-images.githubusercontent.com/32523401/134615391-53d919a8-4396-407f-a0fd-52629a095521.png)
